pythonFiles/vscode_datascience_helpers/dataframes/vscodeDataFrame.py

```diff
@@ -156,12 +160,6 @@ def _VSCODE_getDataFrameInfo(df):
     else:
         columnNames = list(df)
 
-    # Compute the index column. It may have been renamed
```

Comment:
I thought this was necessary in the grid so wondering why it was removed?

Reply:
Part of the data viewer redesign changes included generating an index column, and although we were doing this for our Python variables in the backend, the AML data viewer wouldn't have gotten that numeric column unless their backend changed too. The intent here is to always generate the index column in the UI so that AML will also get it.